.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: build
+on:
+  push:
+    branches:
+    - longhorn*
+  pull_request:
+jobs:
+  build:
+    name: Build Binaries
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

Comment:
Not quite understand why it needs read contents permissions because I don't see it in `.github/workflows/build.yml` of the `longhorn-manager` repo.
Could you elaborate on this?